aggregate_temporal.json

```diff
@@ -235,13 +232,21 @@
         },
         "DistinctDimensionLabelsRequired": {
             "message": "The dimension labels have duplicate values. Distinct labels must be specified."
+        },
+        "TemporalExtentEmpty": {
+            "message": "At least one of the intervals is empty. The second instances in time must always be greater/later than the first instances."
```

Comment:
> The second instances in time must always be greater/later than the first instances.

I don't understand this. Do you mean instants instead of instances?

Reply:
```suggestion
            "message": "At least one of the intervals is empty. The second instant in time must always be greater/later than the first instant."
```